plugins/org.jkiss.dbeaver.ui.editors.connection/src/org/jkiss/dbeaver/ui/preferences/PrefPageProjectNetworkProfiles.java

```diff
@@ -144,12 +144,16 @@ protected DBWNetworkProfile createNewProfile(@Nullable DBWNetworkProfile sourceP
                 getShell(),
                 UIConnectionMessages.pref_page_network_profiles_tool_create_dialog_profile_name,
                 profileName
-            ).trim();
+            );
 
             if (CommonUtils.isEmpty(profileName)) {
                 return null;
             }
 
+            if (CommonUtils.isNotEmpty(profileName)) {
+                profileName = profileName.trim();
+            }
```

Comment:
`CommonUtils.isEmpty` already checks for null, so you can remove the if entirely.

```suggestion
            profileName = profileName.trim();
```